.github/ISSUE_TEMPLATE/new_cheatsheet_proposal.md

```diff
@@ -1,17 +1,32 @@
 ---
-name: new_cheatsheet_proposal
-about: Used to create a proposal to add a new cheat sheet to the project
+name: New Cheat Sheet Proposal
+about: Used to create a proposal to add a new cheat sheet to the project.
 title: New cheat sheet proposal
```

Comment:
Maybe include here placeholder for CS name, similar to what we have for update. E.g.:
`New cheat sheet proposal: [PUT_TARGET_CS_NAME_HERE]`

Reply:
Good call, fixed in 9384911ba05a3719895acc308e528176af92c5cf.